app/src/main/res/values-bn-rBD/strings.xml

```diff
@@ -481,7 +481,7 @@
   <string name="bloodsugarhistory_toolbar_title">%1$s, %2$s, %3$s</string>
   <!-- Summary CV History view -->
   <string name="medicalhistorysummaryview_history">ইতিহাস</string>
-  <string name="medicalhistorysummaryview_diagnosis">রোগ নির্ণয়</string>
+  <string name="medicalhistorysummaryview_diagnosis">রোগ নির্ণয় (প্রয়োজন)</string>
```

Comment:
How did the string resource for this change? Did you copy this from Transifex?

Reply:
I got from new patient medical history

Reply:
What about the strings in other languages?

Ideally, we should *only* change the source file (english), unless we are renaming keys or deleting keys. Adding new strings should go through the standard translation workflow.

Reply:
The text `Diagnosis (Required)` has already been translated. So I am just adding `Required` part to the existing string